ui/shared/components/panel/variants/SavedVariants.jsx

```diff
@@ -76,14 +78,17 @@ class SavedVariants extends React.PureComponent {
     const {
       match, tableState, filters, discoveryFilters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
       tableSummaryComponent, loading, filteredVariantsCount, tagOptions, additionalFilter, updateTableField,
-      variantExportConfig, loadVariants, error,
+      variantExportConfig, loadVariants, error, multiple,
     } = this.props
     const { showAllFilters } = this.state
     const { familyGuid, variantGuid, tag } = match.params
 
-    const appliedTagCategoryFilter = tag || (variantGuid ? null : (tableState.categoryFilter || ALL_FILTER))
+    const tags = tag ? tag.split(TAG_URL_DELIMITER) : tag
+    const tagCategoryFilters = tags || (variantGuid ? [] : [(tableState.categoryFilter || ALL_FILTER)])
+    const firstAppliedTagCategoryFilter = tagCategoryFilters.length > 0 ? tagCategoryFilters[0] : null
+    const appliedTagCategoryFilter = multiple ? tagCategoryFilters : firstAppliedTagCategoryFilter
```

Comment:
this functionality should not be shared, its really page specific and hard to understand whats happening here. Having a `dropdownValue` prop explicitly passed through here would make the page specific logic much clearer